omod-2.0/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_0/UserResource2_0.java

```diff
@@ -49,9 +49,6 @@ public UserAndPassword1_8 save(UserAndPassword1_8 user) {
 		} else {
 			openmrsUser = Context.getUserService().saveUser(user.getUser());
 			Context.refreshAuthenticatedUser();
-			if (openmrsUser.getId() != null && StringUtils.isNotBlank(user.getPassword())) {
-				Context.getUserService().changePassword(openmrsUser, user.getPassword());
-			}
 		}
```

Comment:
Since we're changing this behavior let's make sure to update the docs at https://wiki.openmrs.org/display/docs/REST+Web+Services+API+For+Clients to reflect this